code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -1010,5 +1012,5 @@ Pass a positive integer as an argument to override a bot's default speed.
 		return
 	var/image/I = path[path[1]]
 	if(I)
-		I.icon = null
+		I.icon_state = null

```

Comment:
If someone adds a default icon_state to aibots.dmi, this will set the icon to that instead of removing it. Setting the alpha to 0 is more reliable.